pywebostv/controls.py

```diff
@@ -153,6 +153,11 @@ def request_func():
 
 
 class MediaControl(WebOSControlBase):
+    def list_audio_output_sources(self):
+        sources = ['tv_speaker', 'external_speaker', 'soundbar', 'bt_soundbar', 'tv_external_speaker']
```

Comment:
I just quick google search with the URI you discovered. That led me here: https://github.com/bendavid/aiopylgtv/issues/3#issuecomment-575914925. Turns out there could potentially be more than the 5 items you have in your list. This is why I was curious if we could get this list from the TV itself rather than hard-coding anything. Can you point me to the place where you found this URI (also was it documented somewhere) ?

Reply:
The link you found is where I got this from. Searched quiet a while for a possibility to get the list from the TV itself but could not find anything. So I ended up using the sources I could test with my own TV ('tv_speaker', 'external_speaker', 'soundbar', 'bt_soundbar', 'tv_external_speaker').

Reply:
I think using a hard-coded list like this might lead to issues. What if I use "bt_soundbar" and I don't have a sound bar at all.

Further, in such a case, what is the response of TV when I set_audio_output(..) with that value? If it gives us an error indication of some sort, we will have to confirm that standard_validation is able to take care of it. Or else, we might have to write our own custom validation.

Reply:
You're right, do you have any idea how we could get a list of possible sources from the TV? 

**Output using an existing value:** 
{"type":"response","id":"ec489afb-3d7a-493b-8233-4a65a7dffbe9","payload":{"returnValue":true,"soundOutput":"external_speaker"}}

**Output using non existing value:**
  File "/usr/local/lib/python3.8/site-packages/pywebostv-0.8.4-py3.8.egg/pywebostv/controls.py", line 119, in request_func
OSError: There is No matched extended item: soundOutput

{"type":"error","id":"5a5caa03-9a3c-43f9-aefa-677a4664c07b","error":"500 Application error","payload":{"returnValue":false,"errorText":"There is No matched extended item: soundOutput"}}

Reply:
On the brighter side, good to know the validation format is the same and it works.  I don't know how to list audio sources unfortunately.